Examples/Tests/langmuir/inputs_2d

```diff
@@ -47,16 +47,25 @@ my_constants.k = 2.*pi/20.e-6  # perturbation wavenumber
 # Particles
 particles.species_names = electrons positrons
 
+# Split the loading of the plasma to test multiple injection sources
+# The two sources s1 and s2 are identical except their zmin and zmax
```

Comment:
Could we actually do this in some other test?
This is because the langmuir input file is one of the first test that people run when developing new algorithms, and it would be nice to keep the input script relatively simple.

Reply:
I can remove this test since there is also the nuclear fusion test.